RELEASE/scripts/autoscend/paths/small.ash

```diff
@@ -17,7 +17,30 @@ void small_initializeSettings()
 	{
 		//having vastly lower stats and no easy solutions in hardcore means you always die from flyering
 		//should be replaced with a more elegant solution where detailed estimation / calculation is done.
-		set_property("auto_ignoreFlyer", true);
+		//set_property("auto_ignoreFlyer", true);
+
+		//cap ML to 50 to help avoid getting beaten up
+		int MLCap = 50;
+		string MLSafetyLimit = get_property("auto_MLSafetyLimit");
+		if(MLSafetyLimit == "")
+		{
+			set_property("auto_MLSafetyLimitBackup","empty");
+			set_property("auto_MLSafetyLimit",MLCap);
+		}
+		if(MLSafetyLimit.to_int() > MLCap)
+		{
+			// record existing MLSafetyLimit so it can be restored at end of run
+			set_property("auto_MLSafetyLimitBackup",MLSafetyLimit);
+			set_property("auto_MLSafetyLimit",MLCap);
+		}
+
+		// don't disregard instant karma either. Helps keep ML low
+		string disregardKarma = get_property("auto_disregardInstantKarma");
+		if(disregardKarma == "true")
+		{
+			set_property("auto_disregardInstantKarmaBackup","true");
+			set_property("auto_disregardInstantKarma", "false");
+		}
```

Comment:
If someone changes the ML safety cap or disregardInstantKarma after they've started a run (or even just by opening the relay script before they've run autoscend in the CLI), they will not be updated once autoscend is ran because they are only set in the initialize phase of the run. Do we want to add something where when autoscend is ran, not when the ascension is initialized, this is set?

Reply:
Hmm it's a good point I didn't consider when writing this. I'm leaning towards keeping it as is. Gives flexibility to those who wish to change settings mid run without autoscend forcing them back to what the script thinks is best. Also I like the simplicity of only backing up once and restoring once